src/bundle/Resources/views/themes/admin/ui/component/input_date_and_time.html.twig

```diff
@@ -0,0 +1,36 @@
+{% import '@ibexadesign/ui/component/macros.html.twig' as html %}
+
+{% set is_small = is_small|default(false) %}
+{% set inline_flatpickr = inline_flatpickr|default(false) %}
```

Comment:
I think it's better to avoid using `flatpickr` in the "interface" of our components. Maybe:
```suggestion
{% set is_datetime_popup_inline = is_datetime_popup_inline|default(false) %}
```